cfy_manager/components/influxdb/config/config.toml

```diff
@@ -5,7 +5,7 @@
 # that can be resolved here.
 # hostname = ""
 
-bind-address = "0.0.0.0"
+bind-address = "localhost"
```

Comment:
we could also make this be rendered from the config:
this would have to be replaced with a `utils.files.deploy(src, dst)` call
https://github.com/cloudify-cosmo/cloudify-manager-install/blob/64121b69aa9766b15d71731891fe9af8b9471bb8/cfy_manager/components/influxdb/influxdb.py#L135-L138
and the config line, changed to `bind-address = "{{ influxdb.endpoint_ip }}"`

Reply:
@tehasdf I don't think so. `influxdb.endpoint_ip` has different semantics from `bind-address`.
`influxdb.endpoint_ip` is an empty string when the user wants to install a local influxdb.
If it's non-empty, then we use an existing influxdb.

So if I follow your suggestion, we'll get `bind-address = ""`.

Unless I'm missing something?

Reply:
it won't be empty because we set it to `localhost` in the other change in this PR, but you're right, there's never going to be a reason to set `bind-address` to anything else but localhost

so let's keep as is

Reply:
Actually, `bind-address` will have to be non-`localhost` in one situation: when we use an external InfluxDB (because otherwise, if `bind-address` is `localhost`, then the manager itself won't be able to access it). However, when we use an external InfluxDB, we don't overwrite the `config.toml` file.

So I think the current scheme is good.